README.md

```diff
@@ -6,4 +6,6 @@
 
 This project is based on the idea of using AI to learn strategies in games.
 
-The intention to to model more real world problems, but as a starting point, we're going to work with simple games such as TicTacToe or Connect4 before moving into more complex domains. 
+The intention to to model more real world problems, but as a starting point, we're going to work with simple games such as TicTacToe or Connect4 before moving into more complex domains.
```

Comment:
I think there's a typo in the first line.  "the intention to to" should be "the intention is to"